status.js

```diff
@@ -2,12 +2,24 @@ const Obv = require('obz')
 
 module.exports = function Status() {
   const indexesStatus = {}
+  const indexesLastTime = {}
```

Comment:
You said in an earlier PR that maps would be better when deleting. Should we do that here as well? Otherwise this looks good.

Reply:
Yeah I was thinking about that. If we use a Map, we have to convert it to an object every time we emit the obv. I thought that maybe that's wasteful (object allocation for every obv emit). Gladly the obv emit doesn't happen frequently, so here it was a choice RAM or CPU, and I don't think the CPU will be significantly hurt.

Reply:
Alright, lets leave it like this then. Don't think it will be a problem in practise :)

Reply:
Hmm, maybe it's fine if the obv emits Maps, because this is not crossing any muxrpc boundary

Reply:
Yeah, bigger fish to catch, but this is up for discussion in the future.

Reply:
It's up to you, I'm fine either way.